src/Symfony/Component/Filesystem/Filesystem.php

```diff
@@ -345,8 +345,8 @@ public function symlink($originDir, $targetDir, $copyOnWindows = false)
     /**
      * Given an existing path, convert it to a path relative to a given starting path.
      *
-     * @param string $endPath   Absolute path of target
-     * @param string $startPath Absolute path where traversal begins
+     * @param string $endPath   Path of target
+     * @param string $startPath Path where traversal begins
```

Comment:
Such edge cases are already problematic for the `isAbsolutePath` method because `isAbsolutePath('c:/')` returns `true`, even on unix systems.

If we want to deprecate relative paths, I can add a deprecation notice and revert the doccomment changes.